www/src/components/homepage/homepage-features.js

```diff
@@ -10,7 +10,8 @@ import FuturaParagraph from "../futura-paragraph"
 const HomepageFeatures = () => (
   <div
     sx={{
-      borderBottom: t => `1px solid ${t.colors.ui.border}`,
+      borderColor: `ui.border`,
+      borderBottom: 1,
```

Comment:
Just gave this a spin, and turns out order is important ;-) — this not-really-visible-in-the-screenshot `borderBottom` should render light grey, but uses `intial` as the color, because the `borderBottom` shorthand (`1` expands to `1px solid`) overrides the `borderColor`.

![image](https://user-images.githubusercontent.com/21834/88337100-6a165d00-cd36-11ea-94f3-5d720e39f7ff.png)